setup.cfg

```diff
@@ -5,8 +5,66 @@
 # cds-migrator-kit is free software; you can redistribute it and/or modify it
 # under the terms of the MIT License; see LICENSE file for more details.
 
-[aliases]
-test = pytest
+[metadata]
+name = cds-migrator-kit
+version = attr: cds_migrator_kit.__version__
+
+[options]
+include_package_data = True
+packages = find:
+python_requires = >=3.9
+zip_safe = False
+install_requires =
+    ;    python-ldap>=3.4.0,<3.5.0
+    invenio-app-rdm[opensearch2]==13.0.0b0.dev4
+    invenio-logging[sentry_sdk]>=2.0
+    cds-rdm @ git+https://github.com/CERNDocumentServer/cds-rdm#egg=cds-rdm-1.0.0&subdirectory=site
```

Comment:
installing only site folder as dependency (cds-rdm is not installable on it's own since no setup.cfg is present). We might consider adding it maybe